app/src/main/java/com/github/se/polyfit/ui/screen/OverviewScreen.kt

```diff
@@ -190,7 +191,7 @@ private fun callCamera(
 fun OverviewScreen(
     paddingValues: PaddingValues,
     navController: NavHostController,
-    mealViewModel: MealViewModel
+    mealViewModel: MealViewModel = hiltViewModel()
```

Comment:
does this have to be a param in the OverviewScreen? i.e. when do we plan on passing in a different mealViewModel to overview screen. Otherwise lets just put in the body

Reply:
Yes, you're right. There are two ways to inject viewModels: in the parameters or in the body. The issue with having the injection in the body is that it makes testing exponentially harder. You have to create a custom application and use it during testing. You also have to define many different rules on how the dependencies should be injected. I decided to take the more simplistic approach by directly injecting it as a parameter to reduce the complexity of the code and the tests.